src/typescript/azure-openapi-validator/rules/utilities/resolveNestedSchema.ts

```diff
@@ -94,16 +85,20 @@ export function resolveNestedSchema(schema: SchemaObject): SchemaObject {
       case "null":
         return schemaObject
     }
-    return resolveSchemaObject(schemaObject)
+    const copySchema = copyObject(schemaObject)
+    for (const prop in schemaObject) {
+      if (typeof copySchema[prop] === "object") {
+        copySchema[prop] = resolveSchemaObject(schemaObject[prop])
+      }
+    }
+    return copySchema
```

Comment:
All bug fix code is here? Comments here why ignore the first nested level properties.